nvflare/apis/dxo.py

```diff
@@ -24,7 +24,6 @@ class DataKind(object):
     FL_MODEL = "FL_MODEL"
     WEIGHTS = "WEIGHTS"
     WEIGHT_DIFF = "WEIGHT_DIFF"
-    XGB_MODEL = "XGB_MODEL"
```

Comment:
so we consider the tree-based models as "weights" as well?

Reply:
yes, otherwise the client api will not be able to properly handle it, so using weights as a general term